### PR TITLE
Updating docs, redoing theme

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

Updating the docs

## To run the new template, need to have the following:

`pip install mkdocs-material`
`pip install pymdown-extensions`

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+cd agents-aea/
+pipenv --python 3.7 && pipenv shell
+pip install .[all]
+```
+
+
+## Alternative setup - many errors with dependencies.
+
+``` bash
+pip install cryptography base58 click click-log jsonschema pyyaml google
```
Comment (DavidMinarsch) [inline docs/quickstart.md]: This needs to go please. No longer relevant.

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+## Setup
+
+
+To get started, clone the repo, start a Python environment, and install all dependencies.
+
+``` bash
+git clone git@github.com:fetchai/agents-aea.git
```
Comment (DavidMinarsch) [inline docs/quickstart.md]: please replace with the new approach:

pip install aea

File: docs/tac.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+pipenv --python 3.7 && pipenv shell
+```
+
+
+Install the package.
+``` bash
+python setup.py install
+```
+
+Error at this point.
```
Comment (DavidMinarsch) [inline docs/tac.md]: I am not sure we want this in here. The error should be resolved :)

File: docs/quickstart.md (hunk 2 above)
Comment (DavidMinarsch) [reply docs/quickstart.md]: this installs the basics

for the cli one needs to do: `pip install aea[cli]` and for the whole package `pip install aea[all]`

File: docs/tac.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+git clone git@github.com:fetchai/agents-tac.git --recursive && cd agents-tac
+```
+
+
+Create and launch a virtual environment.
+
+``` bash
+pipenv --python 3.7 && pipenv shell
+```
+
```
Comment (DavidMinarsch) [inline docs/tac.md]: A step is missing here, Diarmid has merged into develop on tac now. Please have a look at the updated main readme there.

File: docs/tac.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Reading https://pypi.org/simple/aea/
+Couldn't find index page for 'aea' (maybe misspelled?)
+Scanning index of all packages (this may take a while)
+Reading https://pypi.org/simple/
+No local packages or working download links found for aea@ git+https://github.com/fetchai/agents-aea.git@develop#egg=aea
+error: Could not find suitable distribution for Requirement.parse('aea@ git+https://github.com/fetchai/agents-aea.git@develop#egg=aea')
+(agents-tac) bash-3.2$ python setup.py install
+```
+
+Install Docker and Docker Compose.
```
Comment (DavidMinarsch) [inline docs/tac.md]: Note, a user also needs to install the docker client I think

File: docs/tac.md (hunk 3 above)
Comment (katharine-murphy) [reply docs/tac.md]: Indeed it is :)

File: docs/tac.md (hunk 5 above)
Comment (katharine-murphy) [reply docs/tac.md]: I'll test this by uninstalling Docker on the next run.

Comment (katharine-murphy) [reply docs/tac.md]: I have added this now.

File: docs/tac.md (hunk 4 above)
Comment (katharine-murphy) [reply docs/tac.md]: Included.

**Pull request merged**